index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Mobile First</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=lato:100,300,400,700,900"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav-bar">
+        <h3 class="my-logo">My Logo</h3>
+        <img class="ham-icon" src="./icons/Union.svg" alt="ham-icon" />
+      </nav>
+      <main>
+        <div class="name-box">
+          <h1 class="main-heading">
+            <span class="heading-top">Hey There.</span>
+            <span class="heading-bottom">I'm Hamza.</span>
+          </h1>
+          <div class="i-am-dev">
+            <h3>I'm a Software Developer</h3>
+          </div>
+
+          <div class="bio">
+            <p class="bio-description">
+              I can help you build a product, feature <br />
+              website Look through some of my <br />
+              work and experience! If you like what you <br />
+              see and have a project you need coded, <br />
+              don’t hesitate to contact me.
+            </p>
+          </div>
+          <div class="icons">
+            <img src="./icons/icon - git.svg" alt="github-icon" />
+            <img src="./icons/Icon - Linkedin.svg" alt="Linkedin-icon" />
+            <img src="./icons/icon-peace.svg" alt="peace-icon" />
+            <img src="./icons/icon-twitter.svg" alt="icon-twitter" />
+            <img src="./icons/icon-medium.svg" alt="icon-medium" />
+          </div>
+        </div>
+      </main>
+
+      <div class="bg-image">
+        <img src="./icons/Illustration.png" alt="image" />
+      </div>
```

Comment:
- [ ] please note that since this image is decorative, it should be added as a background image and not as an img element. You can check this [link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info on why this is convention. Please update this element to be a background image and not an `img` tag thank you.